app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,35 @@
+class MessagesController < ApplicationController
+    skip_before_action :verify_authenticity_token, raise: false
```

Comment:
You shoulnd't need to skip this, it's an important security feature. I assume you were seeing an error, Try removing this and see if it's still happening.